openedx/adg/lms/applications/models.py

```diff
@@ -0,0 +1,122 @@
+"""
+All models for applications app
+"""
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.common.util.date_utils import month_choices, year_choices
+
+
+class ApplicationHub(TimeStampedModel):
+    """
+    Model for status of all required parts of user application submission.
+    """
+    user = models.OneToOneField(User, related_name='application_hub', on_delete=models.CASCADE, )
+    is_prerequisite_courses_passed = models.BooleanField(default=False, )
+    is_application_submitted = models.BooleanField(default=False, )
+    is_assessment_completed = models.BooleanField(default=False, )
+
+    class Meta:
+        app_label = 'applications'
+
+    def __str__(self):
+        return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
+
+
+class UserApplication(TimeStampedModel):
+    """
+    Model for status of all required parts of user application submission.
+    """
+    user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
+    city = models.CharField(verbose_name=_('City'), max_length=255, )
+    organization = models.CharField(verbose_name=_('Organization'), max_length=255, blank=True, )
+    linkedin_url = models.URLField(max_length=255, blank=True, )
```

Comment:
As we are adding in other fields, May we add a translate able verbose_name here `LinkedIn URL`